index.html

```diff
@@ -14,18 +14,23 @@
         <nav class="nav-list">
             
             <a class="logo" href="#">My Logo</a>
+            <ul class="navbar-list">
 
-            <button class="menu-button">
+            <li class="menu-icon">
                 <img class="menu" src="static/images/Union.png" alt=" navigation" />
-            </button>
+            </li>
+            <li class="desktop-nav-list">Portfolio</li>
+            <li class="desktop-nav-list">About</li>
+            <li class="desktop-nav-list">Contact</li>
+          </ul>
 
         </nav>
```

Comment:
- [ ] Good luck aligning your header section. Kindly adjust the height and padding around the navigation elements to match the figma design and also the color around the header 👍🏽. 

Figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/213540082-a98a2a04-d5b3-499a-8c2b-4b2f90b01cc0.png)

Your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/213544179-dcb3a125-c2a0-4a59-9181-a4b5fa714cb7.png)